src/main/kotlin/io/github/mojira/arisa/modules/CommandModule.kt

```diff
@@ -53,14 +62,20 @@ class CommandModule(
         return when (split[0]) {
             // TODO this should be configurable if we move to a registry
             // TODO do we want to add the response of a module via editing the comment?
-            "ARISA_ADD_VERSION" -> addVersionCommand(issue, *arguments)
+            "ARISA_ADD_VERSION" -> {
+                modRestricted = false
```

Comment:
Same as the other PR. Its bad practice to store state globally to controlthis, plus multiple commands will all do whatever the last command was. You can encode this on the return of the module (Have a data class for both username and the boolean for its restricted?)